### PR TITLE
Move default values for options in manifest.json and create config.js…

### PR DESCRIPTION
**Pull request by landryb**
…on (#223)

This way, things will work out of the box much better.
While here, update README.md and fix outdated information.

Would love feedback from @fvanderbiest  :)

### CODE REVIEW

File: addons/cadastrapp/config.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[{
+	"id": "cadastrapp_0",
+	"name": "Cadastrapp",
+	"preloaded": "true",
+	"enabled": "true",
+	"title": {
+		"fr": "Cadastrapp",
+		"en": "Cadastrapp",
```
Comment (fvanderbiest) [inline addons/cadastrapp/config.json:8]: indents are 4 spaces in js, no tabs

File: addons/cadastrapp/manifest.json [hunk 2]
```diff
@@ -24,7 +24,44 @@
...
     ],
-    "default_options": {},
+	"default_options": {
+		"target": "tbar_12",
+		"openToolbarOnLoad": false,
+		"webapp":{
+			"url" : "/cadastrapp/"
+		},
+		"WMSLayer":{
+			"layerNameInPanel":"",
```
Comment (fvanderbiest) [inline addons/cadastrapp/manifest.json:35]: ?

File: addons/cadastrapp/config.json (hunk 1 above)
Comment (landryb) [inline addons/cadastrapp/config.json:8]: Who decided that, so i can hunt him ?

File: addons/cadastrapp/manifest.json (hunk 2 above)
Comment (landryb) [inline addons/cadastrapp/manifest.json:35]: ?? questionning the usecase of this ? By default, if empty, it wont show in the layertree.

**Comment by landryb** [comment]
Fwiw, this branch works fine, except for a small buglet i encountered - as we dont wait for the return of getConfiguration, the 'lookup by owner' features arent added to the toolbar/menus. I'll look into adding them all the time, probably hidden, and unhide them in the success callback from getConfiguration

**Comment by landryb** [comment]
Said buglet fixed by b7a9382 - tested with ffx/ie, logged in/out

**Comment by pierrejego** [comment]
Thank you :) love the Fix scope spaghetti

**Pull request merged**